### PR TITLE
gatsby image conceptual guide

### PR DESCRIPTION
**Pull request by calcsam**

### CODE REVIEW

File: docs/docs/conceptual/using-gatsby-image.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Sending an image with a width of 800px to a  mobile phone with a 400px wide screen means that you are sending unnecessary data over the wire that will, again, delay page load, cause visitors to bounce, and hurt conversions. 
+
+Conversely, some laptops, like Macbook Pros, have a higher pixel-to-density ratio, so need "2x"-size images. 
+
+In order to support responsive images, you need to do the image processing beforehand, as well as generate the markup necessary. Gatsby Image has a `fluid` option that will transform images with a relatively straightforward API.
+
+### Better compression 
+
+The new [WebP image](https://developers.google.com/speed/webp) standard reduces image size by 25-35% for modern browsers. It's possible to support this standard, but the challenge is to also fall back for older browsers that don't support this.
```
Comment (ascorbic) [inline docs/docs/conceptual/using-gatsby-image.md:83]: I wouldn't say it's a challenge (it's pretty simple), just that it needs to be considered.

File: docs/docs/conceptual/using-gatsby-image.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+There are a number of ways to minimize image file size. Below are three of the most common, roughly in order of increasing complexity.
+
+### Cropping and compressing overly large images
+
+A common problem in larger projects is that images are uploaded into a CMS by content operations personnel by teams that may be less aware of appropriate web image sizes and formats. 
+
+For example, a support staff member may take a 1600x2000 pixel screenshot, save it as a PNG, and upload it alongside a helpdesk article. While this is a quite reasonable action, it may degrade page performance significantly. If the article has a 800px maximum width, a 640 x 800 pixel JPG would have displayed at the same quality but a tenth of the size; the extra weight may delay page load by a second or two. 
+
+You can solve this on your own via creating and maintaining a custom image processing pipeline, perhaps during CI/CD, to resize. However, that requires writing, and then maintaining, custom code.  Gatsby Image solves this out of the box; if you use the `width` prop, Gatsby Image will automatically resize larger underlying assets.
```
Comment (ascorbic) [inline docs/docs/conceptual/using-gatsby-image.md:69]:
```suggestion
You can solve this on your own via creating and maintaining a custom image processing pipeline, perhaps during CI/CD, to resize. However, that requires writing, and then maintaining, custom code.  Gatsby Image solves this out of the box; if you use the `width` or `maxWidth` prop, Gatsby Image will automatically resize larger underlying assets.
```

**Review by meganesu — APPROVED** [review]

File: docs/docs/conceptual/using-gatsby-image.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+* **Provide a placeholder during image fetch**. "Progressive images" are image placeholders -- previews of a full image that hold its place during page load time.
+
+* **Minimize image file size to reduce request roundtrip time.** There are a number of ways, from cropping overly large images, to using newer file types, to reduce image file size. 
+
+We built Gatsby Image to provide these things -- a [higher-level building block](https://www.gatsbyjs.com/docs/conceptual/gatsby-core-philosophy/#construct-new-higher-level-web-building-blocks) that has the richness users expect, with the API simplicity developers love (and without maintaining your own image processing pipeline!).
+
+We provide a [detailed guide on using Gatsby Image](docs/how-to/images-and-media/using-gatsby-image/) in the How To section of this documentation. Here, we walk through the question of why these benefits matter, from a user's perspective, and why providing them without Gatsby Image (or something similar) can be difficult. 
+
+## Fetch above the fold images immediately
```
Comment (meganesu) [inline docs/docs/conceptual/using-gatsby-image.md:21]:
```suggestion
## Fetch above-the-fold images immediately 
```

File: docs/docs/conceptual/using-gatsby-image.md (hunk 1 above)
Comment (calcsam) [reply docs/docs/conceptual/using-gatsby-image.md]: 👍 good point

**Comment by calcsam** [comment]
Megan 👍 -ed so I will merge!

**Pull request merged**